City-of-Helsinki/drupal-helfi-rekry
# Uhf 7765 jobs search translations

# [UHF-0000](https://helsinkisolutionoffice.atlassian.net/browse/UHF-0000)
Added missing translations to job search app.

## What was done
* Added translation files + translations
* Fixd some contexts + english string in code
* Ninja fixed sorting results

## How to install

Check instructions here: https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/150 (All the same steps apply here, but **remember to change branch name** to UHF-7765-jobs-search-translations)

## How to test
* Translate the landing page with `job_search` paragraph to all languages (fi, en. sv) and check that no UI translation are missing
* Check that sorting results works (and updates url params when changing selection)

## Designers review
